binary/reader.js

```diff
@@ -52,6 +52,26 @@ goog.require('jspb.BinaryConstants');
 goog.require('jspb.BinaryDecoder');
 goog.require('jspb.utils');
 
+/**
+ * Whether to enforce that string fields are valid utf8.
+ *
+ * <p>Currently set to `ALWAYS`, can be set to `DEPRECATED_PROTO3_ONLY` to only
```

Comment:
for OSS this is a breaking change im guessing

probably fine but im also guessing that our goog.defines are not very usable...

just make sure this is mentioned in the commit message